packages/icon/src/create-icon.tsx

```diff
@@ -32,10 +33,14 @@ export function createIcon(options: CreateIconOptions) {
   const {
     viewBox = "0 0 24 24",
     d: pathDefinition,
-    path,
     displayName,
     defaultProps = {},
   } = options
+  let path = options.path
+
+  if (Array.isArray(path)) {
+    path = path.map((p) => React.cloneElement(p, { key: uuid() }))
```

Comment:
I'd refrain from adding an additional dependency for this for these reasons:
- the `path` property of the icon is highly unlikely to change. Using the array index ( `path.map((p, index) => cloneElement(p, { key: index }))` is sufficient here
- although unlikely, if `createIcon` is called multiple times, e.g. because its within the body of a rerendering component, `uuid` would re-execute and create a _new_ key for the _same_ path, which defeats the purpose of using a key in the first place.